src/Commando/Command.php

```diff
@@ -486,11 +545,26 @@ public function parse()
             // Used in the \Iterator implementation
             $this->sorted_keys = array_keys($this->options);
             natsort($this->sorted_keys);
+
```

Comment:
Is there a reason this is still here?

Reply:
I based my branch off of master, but I am merging to the dev branch in an attempt to get it up to date again, so people will actually use it for PRs, as the README suggests.

Anyway. This commented out section of code exists in master, but not in dev, and is not actually part of the changes related to `reduce`